includes/admin/class-wc-admin-setup-wizard.php

```diff
@@ -578,55 +443,257 @@ protected function wc_setup_wcs_tout() {
+	 * @param bool   $background Optional. Whether or not to queue the install in the backgroud.
+	 */
+	protected function install_plugin( $plugin_id, $plugin_info, $background = true ) {
+		if ( ! empty( $plugin_info['file'] ) && is_plugin_active( $plugin_info['file'] ) ) {
+			return;
+		}
+
+		if ( $background ) {
+			wp_schedule_single_event( time() + 10, 'woocommerce_plugin_background_installer', array( $plugin_id, $plugin_info ) );
+		} else {
+			WC_Install::background_installer( $plugin_id, $plugin_info );
+		}
+	}
+
+	/**
+	 * Helper method to install Jetpack.
+	 *
+	 * @param bool $now Optional. Whether or not to queue the install.
+	 */
+	protected function install_jetpack( $now = false ) {
+		$this->install_plugin( 'jetpack', array(
+			'file'      => 'jetpack/jetpack.php',
+			'name'      => __( 'Jetpack', 'woocommerce' ),
+			'repo-slug' => 'jetpack',
+		), ! $now );
+
+		if ( ! $now ) {
+			update_option( 'woocommerce_setup_queued_jetpack_install', true );
+		}
+	}
+
+	/**
+	 * Helper method to install WooCommerce Services and its Jetpack dependency.
+	 */
+	protected function install_woocommerce_services() {
+		$this->install_jetpack();
+		$this->install_plugin( 'woocommerce-services', array(
+			'file'      => 'woocommerce-services/woocommerce-services.php',
+			'name'      => __( 'WooCommerce Services', 'woocommerce' ),
+			'repo-slug' => 'woocommerce-services',
+		) );
+	}
+
+	/**
+	 * Get the WCS shipping carrier for a given country code.
+	 *
+	 * Can also be used to determine if WCS supports a given country.
+	 *
+	 * @param $country_code
+	 * @return bool|string Carrier name if supported, boolean False otherwise.
+	 */
+	protected function get_wcs_shipping_carrier( $country_code ) {
+		switch ( $country_code ) {
+			case 'US':
+				return 'USPS';
+			case 'CA':
+				return 'Canada Post';
+			default:
+				return false;
+		}
+	}
+
+	/**
+	 * Get shipping methods based on country code.
+	 *
+	 * @param $country_code
+	 * @return array
+	 */
+	protected function get_wizard_shipping_methods( $country_code ) {
+		$shipping_methods = array(
+			'live_rates' => array(
+				'name'        => __( 'Live Rates', 'woocommerce' ),
+				'description' => __( 'Shipping rates updated in realtime. Powered by Jetpack.', 'woocommerce' ),
+			),
+			'flat_rate' => array(
+				'name'        => __( 'Flat Rate', 'woocommerce' ),
+				'description' => __( 'Set a fixed price to cover shipping costs.', 'woocommerce' ),
+				'settings'    => array(
+					'cost' => array(
+						'type'        => 'text',
+						'description' => __( 'What would you like to charge for flat rate shipping?', 'woocommerce' ),
+					),
+				),
+			),
+			'free_shipping' => array(
+				'name'        => __( 'Free Shipping', 'woocommerce' ),
+				'description' => __( "Don't charge for shipping.", 'woocommerce' ),
+			),
+		);
+
+		$live_rate_carrier = $this->get_wcs_shipping_carrier( $country_code );
+
+		if ( false === $live_rate_carrier ) {
```

Comment:
We should check here too if `current_user_can('install_plugins')`, for consistency, the same way you did for the payment methods.